amazon-cloudwatch/daemonset.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: cloudwatch-agent
+  namespace: amazon-cloudwatch
+spec:
+  selector:
+    matchLabels:
+      name: cloudwatch-agent
+  template:
+    metadata:
+      labels:
+        name: cloudwatch-agent
+    spec:
+      containers:
+        - name: cloudwatch-agent
+          image: amazon/cloudwatch-agent:latest
+          imagePullPolicy: Always
+          #ports:
```

Comment:
Any reason this is commented out? Should we remove this completely?

Reply:
This is how the instructions in aws had it :/ so not sure. I can remove it if you think it shouldn't be there.

Reply:
No, let's leave it then. It makes more sense to be consistent with their docs/original files I suppose. 👍